code/game/machinery/cryo.dm

```diff
@@ -1,65 +1,116 @@
 #define HEAT_CAPACITY_HUMAN 100 //249840 J/K, for a 72 kg person.
 
-/obj/machinery/atmospherics/unary/cryo_cell
+/obj/machinery/cryo_cell
 	name = "cryo cell"
 	icon = 'icons/obj/machines/cryogenics2.dmi'
 	icon_state = "cell-off"
 	density = 0
 	anchored = 1.0
 	layer = BELOW_OBJ_LAYER
 
-	temperature = 100
-	gas_type = GAS_TYPE_OXYGEN
+	var/temperature = 100
 
-	var/on = 0
+	var/on = FALSE
+	var/auto_release = FALSE
+	var/release_notice = FALSE
 	use_power = 1
 	idle_power_usage = 20
 	active_power_usage = 200
+	var/idle_ticks_until_shutdown = 60 //Number of ticks permitted to elapse without a patient before the cryotube shuts itself off to save processing
 
 	var/mob/living/carbon/occupant = null
 	var/obj/item/reagent_container/glass/beaker = null
 
-/obj/machinery/atmospherics/unary/cryo_cell/New()
-	..()
-	initialize_directions = dir
-	start_processing()
+/obj/machinery/cryo_cell/New()
+
+	return ..()
```

Comment:
If the proc does nothing but call the parent proc it may as well not exist.

Reply:
Ah, forgot to delete that after I retooled the radio notification system.